tests/_plugins/ui/_impl/test_input.py

```diff
@@ -75,6 +84,18 @@ def test_slider_invalid_bounds() -> None:
     assert "Invalid bounds" in str(e.value)
 
 
+def test_slider_exclusive_args() -> None:
```

Comment:
can we add a test for empty `steps` (not sure how we want to handle) and invalid bounds (value is not in steps)